Concepts docs and new FCL Quickstart (#323)
onflow/flow: docs/content/index.mdx

import { MenuTitle } from "./components/menu";
import { theme } from "gatsby-theme-flow/src/colors";

> Ready to start building? Take a tutorial, or have a look at an example application.

Reviewer: ```suggestion
Ready to start building? Take a tutorial or have a look at an example application.
```

### 🏃‍♂️ App Quickstart Using React

Follow the [Flow App Quickstart](/flow-js-sdk/flow-app-quickstart/) for a detailed guide covering how to use the [Flow JS SDK](/flow-js-sdk) to build a serverless app that connects to Flow, using React!

> ### 😸 Kitty-Items Full-Stack Example App

Reviewer: ```suggestion
### 😸 Kitty Items Full-Stack Example App
```

> The Kitty Items application is a full-featured NFT marketplace that demonstrates what a complex full-stack application

Reviewer: ```suggestion
The Kitty Items application is a full-featured NFT marketplace that demonstrates what a full-stack application can look like on Flow.
```

built on Flow will look like. It's designed to showcase the power of Cadence and Flow's client libraries.

> Have a look at the [Kitty Items Github repository](https://github.com/onflow/kitty-items) for more details.

Reviewer: ```suggestion
Have a look at the [Kitty Items GitHub repository](https://github.com/onflow/kitty-items) for more details.
```
